### PR TITLE
api changes for UI

### PR DESCRIPTION
**Pull request by derek-ho**
### Description
[Describe what this change achieves]

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: schema/system/integration-put.schema
Comment (YANG-DB) [inline schema/system/integration-put.schema:1]: why do we need an integration-put ? can't we just use the integration-instance for put operation as-well ?

File: docs/API/swagger.yaml [hunk 1]
```diff
@@ -446,21 +480,12 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '../../schema/system/integration-instance.schema'
+              $ref: '../../schema/system/integration-put.schema'
```
Comment (YANG-DB) [inline docs/API/swagger.yaml:483]: why do we need a dedicated resource for the put operation ? it is not a conventional  REST operation

**Review by YANG-DB — CHANGES_REQUESTED** [review]

File: schema/system/integration-put.schema
Comment (YANG-DB) [inline schema/system/integration-put.schema:1]: please remove this type - it is identical to the integration instance

**Review by YANG-DB — CHANGES_REQUESTED** [review]

File: docs/API/swagger.yaml [hunk 2]
```diff
@@ -416,20 +445,25 @@ paths:
...
-                  version: 0.1.0
-                  description: Nginx HTTP server collector for east cost prod systems
-                  template: https://github.com/opensearch-project/observability/blob/2.x/integrations/nginx/config.json
-                  creationDate: '2016-08-29T09:12:33.001Z'
-                  status: LOADED
-                  assets:
-                    - name: sso-logs-dashboard-new.ndjson
-                      creationDate: "'2016-08-29T09:12:33.001Z'"
-                      status: LOADED
+              examples:
```
Comment (derek-ho) [inline docs/API/swagger.yaml:448]: here

**Comment by Swiddis** [comment]
@derek-ho what steps do we need to do in order to get this revised and merged? We really should be freezing our data interfaces soon.

**Comment by derek-ho** [comment]
Closing, as we went forward with frontend plugin design

**Pull request closed without merging**